public/votes.cfg

```diff
@@ -51,24 +43,10 @@ add_vote "1000" "sv_scorelimit 1000"
 add_vote "[===] timelimit [===]" "sv_map"
 
 add_vote "timelimit 0" "sv_timelimit 0"
+add_vote "timelimit 5" "sv_timelimit 5"
 add_vote "timelimit 10" "sv_timelimit 10"
 add_vote "timelimit 15" "sv_timelimit 15"
 add_vote "timelimit 20" "sv_timelimit 20"
+add_vote "timelimit 30" "sv_timelimit 30"
 
-exec cfg/share/votes_slots.cfg
-
-add_vote "[===] game [===]" "sv_map"
-
-add_vote "[ ] enable ready mode" "sv_player_ready_mode 1;say 0.7 players can now pause the game with f1 ready_change. And 0.6 players with the /ready chat command."
-add_vote "[ ] disable ready mode" "sv_player_ready_mode 0;say the ready_change command no longer allows pausing the game"
-add_vote "pause" "pause_game"
-add_vote "restart" "restart 10"
-add_vote "shuffle teams" "shuffle_teams"
-add_vote "swap teams" "swap_teams"
-add_vote "reload server" "reload"
-
-add_vote "[===] gametype [===]" "sv_map"
-
-add_vote "CTF" "sv_gametype ctf;reload"
-add_vote "DM" "sv_gametype dm;reload"
```

Comment:
I see that all maps force the gametype now. But do we really want to disallow voting DM on CTF maps?